tests/integration/helpers.bash

```diff
@@ -197,6 +197,15 @@ function set_resources_limit() {
   sed -i 's/\("linux": {\)/\1\n   "resources": { "pids": { "limit": 100 } },/'  "$bundle/config.json"
 }
 
+# Helper function to make /sys/fs/cgroup writable
+function set_cgroup_mount_writable(){
```

Comment:
nit: space before `{`.

I think there is another potential user of this function in commit 82fa19417986da834b7d6bdd7d3539c37a5735a4, maybe you can also make it use it while at it?

Reply:
done